client/app/utils/i18n.js

```diff
@@ -42,6 +42,34 @@ if (isServer()) {
 
 const I18n = require('i18n-js'); // eslint-disable-line no-undef
 
+function initialize(railsContext, env) {
+  I18n.locale = railsContext.i18nLocale;
+  I18n.defaultLocale = railsContext.i18nDefaultLocale;
+
+  if (env === 'development') {
+    I18n.missingTranslation = function throwMissingTranslation(scope) {
+      throw new Error(`Missing translation: ${scope}`);
```

Comment:
It seems to throw in `development`, unlike the PR title promised. I did see the "in progress" label, so if this is just for ease of development then nevermind, just remember to flip it before merging!